src/Command/InfectionCommand.php

```diff
@@ -69,7 +69,7 @@ protected function configure()
                 'test-framework',
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Name of the Test framework to use (phpunit, phpspec)',
+                'Name of the Test framework to use (phpunit, phpspec, codeception)',
```

Comment:
could you please use `TestFrameworkTypes::TYPES` constant here as we do in the ConfigureCommand?